api/server/http-rest/server.go

```diff
@@ -0,0 +1,112 @@
+package http_rest
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v5/api/server/middleware"
+	"github.com/prysmaticlabs/prysm/v5/runtime"
+)
+
+var _ runtime.Service = (*Server)(nil)
+
+// restHandler is a functional interface that implements the rest handler functionality.
+type restHandler func(
+	h http.HandlerFunc,
+	w http.ResponseWriter,
+	req *http.Request,
+)
+
+// Config parameters for setting up the http-rest service.
+type config struct {
+	httpAddr       string
+	allowedOrigins []string
+	muxHandler     restHandler
+	router         *mux.Router
+	timeout        time.Duration
+}
+
+// Server serves HTTP JSON traffic.
```

Comment:
```suggestion
// Server serves HTTP JSON traffic.
```
That's not entirely true because we also serve SSZ. As mentioned earlier, I don't see a good reason to document this unless there is something non-obvious that you want to explain.